test/API.spec.js

```diff
@@ -0,0 +1,14 @@
+const template = require('../packages/oc-template-handlebars');
+const compiler = require('../packages/oc-template-handlebars-compiler');
```

Comment:
technically it would be better to have 2 separate spec files but maybe too much 😄